### ISSUE TITLE
MCT should build out-of-source

### ISSUE THREAD

**Issue by quantheory** [MCSclimate/MCT#9]
Right now, CESM (and presumably ACME) has to copy several files from the source tree after running configure, in order to build out-of-source. I'm not sure what the best way is to address this using autoconf, but it seems like either the path to these files needs to be changed somewhere (from relative to absolute?), or the configure script should copy these files.

**Comment by rljacob** [comment]
Are you referring to the Makefile and Makefile.conf files that are copies from the MCT source to the bld/intel/mpi-serial/nodebug/nothreads/mct or similar directory?   The mct.buildlib script used to copy everything but that was eliminated at some point (prior to beta10).

**Comment by quantheory** [comment]
@rljacob Yes, I think that we're mostly on the same page. The point is that it would be good to be able to check out MCT by itself (without some larger model) and be able to do out-of-source builds. The `buildlib.mct` in CESM's Machines directory does this now, but it's not part of MCT. I don't think that changing this would be that difficult, but I don't know autotools well enough to propose a specific change.
